src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        Random random = new Random(Color.values().length);
+        return Color.values()[random.nextInt()];
```

Comment:
The `Random` object should not be initialized with a seed. Instead, use `Random random = new Random();` and modify `random.nextInt()` to `random.nextInt(Color.values().length)` to ensure it selects a valid index within the bounds of the `Color` enum.